Sources/BraintreeCore/BTConfiguration+Extensions.swift

```diff
@@ -36,6 +36,11 @@ import Foundation
         json?["payWithVenmo"]["environment"].asString()
     }
 
+    /// Indicates whether ECD is enabled for the Venmo merchant.
+    var isVenmoEnrichedCustomerDataEnabled: Bool {
+        json?["payWithVenmo"]["ecdEnabled"].isTrue ?? false
+    }
+
```

Comment:
Heads up that this will need to move into the `BTConfiguration+Venmo` file once this PR is merged in: https://github.com/braintree/braintree_ios/pull/1032

Reply:
done!